webapp/channels/src/utils/constants.tsx

```diff
@@ -1088,6 +1088,7 @@ export const HostedCustomerLinks = {
 
 export const DocLinks = {
     ABOUT_TEAMS: 'https://docs.mattermost.com/welcome/about-teams.html#team-url',
+    ADVANCED_LOGGING: 'https://mattermost.com/pl/advanced-logging',
```

Comment:
TOOD: Setup the link

Reply:
@DHaussermann :point_up: FYI

Reply:
@hanzei I'm confused. 
The link on the page does in fact point there 
![image](https://github.com/mattermost/mattermost/assets/35588783/13fdb5f9-2425-4d73-9746-ef161d82b595)

But there is no page to serve at `https://mattermost.com/pl/advanced-logging' 
![image](https://github.com/mattermost/mattermost/assets/35588783/86bcaa09-8b87-48ca-9653-1a22e55bb432)

Is this something we need help with? @cwarnermm may be able to help.

Reply:
Thanks, @DHaussermann! The advanced-logging permalink doesn't appear in [Marketing's master list](https://docs.google.com/spreadsheets/d/1O601H_A0IM8pR3FOfue29_C8flGV7xK3ts-tJUVDHHg/edit#gid=0), which suggests that it's not defined. It should take users to https://docs.mattermost.com/comply/audit-log.html

@emdecr - Is this something you can help with?

Reply:
@DHaussermann - For context, all in-product links to docs or other educational assets now use a permanent link created and maintained by Marketing. The Marketing channel includes a link to the [master spreadsheet](https://docs.google.com/spreadsheets/d/1O601H_A0IM8pR3FOfue29_C8flGV7xK3ts-tJUVDHHg/edit#gid=0) of permalinks.

We moved to this approach to ensure the in-product links were more stable, available, and robust (compared to absolute docs links).

If you ever see a perma link that isn't resolving like this one, it's likely just not yet set up in the Marketing system where they're maintained. The disconnect can happen because the link management and the conversion of in-product links are/were separate/distinct lifts by different teams.

Reply:
@cwarnermm Thanks for the flag and knowledge share. Added `/pl/advanced-logging` to [the sheet](https://docs.google.com/spreadsheets/d/1O601H_A0IM8pR3FOfue29_C8flGV7xK3ts-tJUVDHHg/edit?disco=AAAA_B8b34c) and set up the redirect. Thx y'all!

Reply:
Thanks!!

Reply:
Thanks @cwarnermm and @emdecr for the help and knowledge share 👍